content/en/client/authorized.md

```diff
@@ -35,9 +35,15 @@ https://mastodon.example/oauth/authorize
 
 Note the following:
 
-* `client_id` was obtained when registering our application.
-* `scope` must be a subset of our registered app's registered scopes. It is a good idea to only request what you need. See [OAuth Scopes]({{< relref "api/oauth-scopes" >}}) for more information.
-* `redirect_uri` is one of the URIs we registered with our app. We are still using "out of band" for this example, which means we will have to manually copy and paste the resulting code, but if you registered your application with a URI that you control, then the code will be returned as a query parameter `code` and can be logged by your request handler. See the response section of the API method documentation for more information on this.
+- `client_id` was obtained when registering our application.
+- `scope` must be a subset of our registered app's registered scopes. It is a good idea to only request what you need. See [OAuth Scopes]({{< relref "api/oauth-scopes" >}}) for more information.
+- `redirect_uri` is one of the URIs we registered with our app.
+
+We are still using "out of band" for this example, which means we will have to manually copy and paste the resulting code, but if you registered your application with a URI that you control, then the code will be returned as a query parameter `code` by your request handler for the redirect URI. See the response section of the API method documentation for more information on this.
+
+{{< hint style="warning" >}}
+Treat the `code` query parameter as if it were a password, you should ensure that it is not logged in request logs.
+{{< /hint >}}
```

Comment:
This displays as follows: <img width="805" alt="Screenshot 2024-05-15 at 20 26 06" src="https://github.com/mastodon/documentation/assets/30827/7b9c9c69-2b0c-4cb4-a3c5-33a65e751bc5">